sw/host/opentitanlib/src/proxy/protocol.rs

```diff
@@ -280,6 +280,9 @@ pub enum ProxyRequest {
     RemovePinStrapping {
         strapping_name: String,
     },
+    ApplyDefaultConfiguration {
```

Comment:
Maybe call this `ApplyDefaultConfigurationWithStrapping`.

Reply:
Done

Reply:
I am a bit confused. Why is this renamed `ApplyDefaultConfigurationWithStrapping`? If I understand the code correctly, `Request::ApplyDefaultConfigurationWithStrapping` does not actually apply a strapping, whereas `ProxyRequest::ApplyDefaultConfigurationWithStrapping` does. Why do we have two `ApplyDefaultConfigurationWithStrapping` ?

Reply:
Sorry about that, I got a little too excited about find and replace. Only the appropriate fields, i.e. `ProxyResponse` and `ProxyRequest`, have been renamed to `ApplyDefaultConfigurationWithStrapping` now.